components/Archetype/basic_usage.rst

```diff
@@ -0,0 +1,29 @@
+Basic Usage
+===========
+
+.. code-block:: php
+
+    // File example: src/script.php
+    <?php
+
+    // update this to the path to the "vendor/" directory, relative to this file
+    require_once __DIR__ . '/vendor/autoload.php';
+
+    use Sylius\Component\Archetype\Model\Archetype;
+    use Sylius\Component\Product\Model\Product;
```

Comment:
I'm not sure if we should use `Product` component in this example. 

Maybe instead of `new Product()` we should write `new ClassThatImplementsArchetypeSubjectInterface()`?
In the comment we can add 'e.g. `Sylius\Component\Product\Model\Product` from Product component', WDYT?